frontend/src/pages/Inbox/components/QuickReply.js

```diff
@@ -57,19 +52,12 @@ const QuickReply = ({
           selectWebpageType(
             interceptExercise,
             releaseCodes,
-            showWebpage,
+            (...args) => dispatch(showWebpage(...args)),
```

Comment:
don't need args here. Can just pass dispatch(showWebpage) and it'll pass the args implicitly 
```suggestion
            dispatch(showWebpage),
```

Reply:
No we can't. Rui and I spent some time on this problem e that way is the way (maybe not so correct) to make it work.

Reply:
good point but instead of using `args` we can pass an actual argument
```
webPage => dispatch(showWebpage(webPage))
```